melusine/nlp_tools/lemmatizer.py

```diff
@@ -0,0 +1,101 @@
+import logging
+import os
+from typing import Union
+
+import numpy
+import spacy
+from spacy.language import Language
+from spacy_lefff import LefffLemmatizer, POSTagger, Downloader
+
+from melusine.utils.verbitim_transformer import VerbitimTransformer
+
+
+@Language.factory('melt_tagger')
+def create_melt_tagger(nlp, name):
```

Comment:
Hello, could you add docstring to understand what the following functions do?
Best,

Reply:
+ type hinting for every function with format compatible for Python <3.9

def ma_fonction(malist: List[int], mastring: str) -> Union[List, str]: